TCAT/Controllers/InformationViewController.swift

```diff
@@ -351,3 +296,106 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
     }
 
 }
+
+// MARK: Table View Data Source
+extension InformationViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return content.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return content[section].count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        // Initalize and format cell
+        let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Cells.informationCellIdentifier, for: indexPath)
+        cell.selectionStyle = .none
+        cell.backgroundColor = Colors.white
+        cell.textLabel?.textColor = Colors.primaryText
+        cell.textLabel?.font = .getFont(.regular, size: 14)
+        cell.textLabel?.textAlignment = .center
+
+        // Set cell content
+        cell.textLabel?.text = content[indexPath.section][indexPath.row].name
+
+        // Set custom formatting based on cell
```

Comment:
A `switch` statement seems like overkill here, since you're only checking one case?

Reply:
agree unless you plan on adding more cases in the near future